packages/react-scripts/config/config.js

```diff
@@ -0,0 +1,110 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+const paths = require('./paths');
+
+const appDirectory = fs.realpathSync(process.cwd());
```

Comment:
Anywhere that uses this should probably use `paths.appPath` instead.

Reply:
Implemented!